.vscode/licence.code-snippets

```diff
@@ -0,0 +1,21 @@
+{
+  "Lisence shortcut": {
+    "prefix": "lisence",
+    "body": [
+      "/*",
+      " * KIELER - Kiel Integrated Environment for Layout Eclipse RichClient",
+      " *",
+      " * http://rtsys.informatik.uni-kiel.de/kieler",
+      " *",
+      " * Copyright 2021 by",
```

Comment:
How cool is that: https://chaseadams.io/posts/using-current-date-and-time-in-vs-code-snippets/

Did not know it works. But it works. However, of course it only works on insert. It will not about every license header automatically. For that a codemod would be required.